pack/fhv/fhvc.json

```diff
@@ -283,6 +283,174 @@
+    "code": "10686",
+    "double_sided": true,
+    "doom": 6,
+    "encounter_code": "day_of_the_feast",
+    "encounter_position": 1,
+    "faction_code": "mythos",
+    "flavor": "The people of the Vale dance with an almost manic energy. Riotous music and a sweet perfume scent hang in the air.",
+    "illustrator": "Rob Laskey",
+    "name": "All is Full of Love",
+    "pack_code": "fhvc",
+    "position": 186,
+    "quantity": 1,
+    "stage": 1,
+    "text": "<b>Forced</b> – When you draw a weakness: Cancel its revelation effect and discard it.",
+    "type_code": "agenda"
+  },
+  {
+    "back_name": "The Bliss",
+    "back_text": "<b>If you flipped this act while or after playing the Dawn of the Final Day prelude, read the following</b>\n<blockquote><i>you cannot say why but you know why you are here you are here to watch to see to know to perceive to feel to understand to hold in your heart the truth of the cosmos which is that you are but a mote of dust in the brilliant light of infinity and that a single puff of air will send you spiraling into oblivion a single moment and you will cease to be if you even ever were why did you look into the brilliance why did you stare so long at the shining blinding brightness of the stars</i></blockquote>\nYou must either duffer 1 mental trauma or search the collection for a random basic [[Madness]] weakness and add it to your deck. It does not count toward your deck size. You earn 2 bonus experience.",
+    "code": "10687",
+    "clues": null,
+    "double_sided": true,
+    "encounter_code": "day_of_the_feast",
+    "encounter_position": 2,
+    "faction_code": "mythos",
+    "flavor": "A beautiful sheen hangs over everything, shimmering in the air around you.",
+    "illustrator": "Rob Laskey",
+    "name": "Dawn of the Final Day",
+    "pack_code": "fhvc",
+    "position": 187,
+    "quantity": 1,
+    "stage": 1,
+    "text": "<b>Objective</b> – Talk to the locals and prepare yourself for the Feast.",
+    "type_code": "act"
+  },
+  {
+    "back_name": "The Sleep",
```

Comment:
`"back_flavor": "Glowing spores hang in the air around the effigy, lighting up the mirrors and bathing the festival in a faerie glow. You breathe in and feel your mouth and throat buzz with a light, tingling sensation. The last thing you hear as you pas out is the sound of lilting flutes and tambourines.",`

Reply:
To go above back_name